modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemander.java

```diff
@@ -687,6 +697,8 @@ public void handleSupplyMessage(
 
             final GridDhtPartitionTopology top = grp.topology();
 
+            grp.metrics().onRebalanceBatchReceived(supplyMsg.messageSize());
```

Comment:
Let's rename `onRebalanceBatchReceived` with a `addRebalancingReceivedBytes`. It's more clear to read such callbacks.